examples/platform/silabs/efr32/rs911x/hal/efx_spi.c

```diff
@@ -342,11 +372,12 @@ int16_t rsi_spi_transfer(uint8_t * tx_buf, uint8_t * rx_buf, uint16_t xlen, uint
     */
     if (xlen <= MIN_XLEN || (tx_buf == NULL && rx_buf == NULL))
     {
-        return RSI_ERROR_INVALID_PARAM;
+        rx_buf = (uint8_t *)&dummy_buffer;
```

Comment:
In what case is there a transfer with no TX nor RX data expected? 
I would also expect that xlen should be at a minimal set to the size of dummy_buffer.
This seems wrong.